deploy/readme.md

```diff
@@ -0,0 +1,32 @@
+# Deploy 
+
+## Deploy Components
+
+Reusable components for deployment and/or data migration. 
+
+### Migrate 
+  Base instructions used for data migration per environment using flyway and Postgres.  
+#### Example
+```yaml 
+  migrate-int:
+  extends: .migrate
+  stage: migrate-int
+```
+
+### Deploy
+
+Base instructions used for deployment. 
+  
+#### Example
+```yaml 
+deploy-int:
```

Comment:
You've left out the variables in your examples, it won't work without them will it?